torch_geometric/loader/neighbor_loader.py

```diff
@@ -475,18 +487,23 @@ def to_index(tensor):
         if isinstance(input_nodes, Tensor):
             return None, to_index(input_nodes)
 
+        # Can't infer number of nodes from a group_name; need an attr_name
         if isinstance(input_nodes, str):
-            num_nodes = feature_store.get_tensor_size(input_nodes)[0]
-            return input_nodes, range(num_nodes)
+            raise NotImplementedError(
```

Comment:
Why do we raise here instead of fix it? Shouldn't `feature_store.get_tensor_size(TensorAttr(group_name=input_nodes))[0]` fix this?

Reply:
That get tensor size call wouldn't work as intended since get tensor rise needs both a group name and are name, hence the raise

Reply:
So what we could do is to select the first `TensorAttr` that has `group_name==input_nodes`, right? But yeah, not ideal.